app/src/main/java/org/oppia/android/app/topic/TopicActivity.kt

```diff
@@ -49,13 +50,18 @@ class TopicActivity :
       TOPIC_ACTIVITY_PARAMS_KEY,
       TopicActivityParams.getDefaultInstance()
     )
+    classroomId = checkNotNull(
+      args?.classroomId
+    ) {
+      "Expected classroom ID to be included in intent for TopicActivity."
+    }
```

Comment:
Nit: Please reformat this and topicId below:
```suggestion
    classroomId = checkNotNull(args?.classroomId) {
      "Expected classroom ID to be included in intent for TopicActivity."
    }
```

Reply:
Done.